product_docs/docs/biganimal/release/free_trial/detail/experiment/index.mdx

```diff
@@ -10,6 +10,7 @@ Here are a few examples to get you started:
 
 * [Import data](import_data)
 * [Backup and restore](backup_and_restore)
+* [Linking to your Oracle database](../../../getting_started/creating_a_cluster/cloud_infra/oracle_dblink/)
```

Comment:
I would add this as a section and point to the topic from the section. In the section you can explain why you might want to do this.

Reply:
Addressed the first comment.

For the second comment, I feel it's better to reference the Cluster networking architecture page in point 7 as it covers information about both private and public clusters and further links to the new parent topic. I might have not understood your suggestion completely and if that is the case feel free to edit the file.